modules/core/src/main/java/org/apache/ignite/internal/hlc/ClockWaiter.java

```diff
@@ -49,7 +46,7 @@
  * no SafeTime mechanisms are involved.
  */
 public class ClockWaiter implements IgniteComponent {
-    private static final IgniteLogger LOG = Loggers.forClass(ClockWaiter.class);
+    private final IgniteLogger log = Loggers.forClass(ClockWaiter.class);
```

Comment:
Ok, but why? Does this have something to do with thin clients?

Reply:
Nope, but the `core` module is a dependency of the thin client, so the client restrictions are effective for the `core` module. Ideally, we either need something like `server-core` module. It seems too much to introduce a module just for the clock-related code.

Reply:
Sure, I'm not asking you to move modules